DigitalLearningSolutions.Web/Controllers/HomeController.cs

```diff
@@ -1,12 +1,69 @@
-﻿using Microsoft.AspNetCore.Mvc;
-
-namespace DigitalLearningSolutions.Web.Controllers
-{
-    public class HomeController : Controller
-    {
-        public IActionResult Index()
-        {
-            return View();
-        }
-    }
-}
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace DigitalLearningSolutions.Web.Controllers
+{
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.ViewModels.Common.MiniHub;
+    using DigitalLearningSolutions.Web.ViewModels.Home;
+    using Microsoft.Extensions.Configuration;
+
+    public class HomeController : Controller
+    {
+        private readonly List<MiniHubSection> sections = new List<MiniHubSection>(
+            new[]
+            {
+                new MiniHubSection { ControllerName = "Home", ActionName = "Welcome", SectionTitle = "Welcome" },
+                new MiniHubSection { ControllerName = "Home", ActionName = "Products", SectionTitle = "Products" },
+                new MiniHubSection
+                    { ControllerName = "Home", ActionName = "LearningContent", SectionTitle = "Learning Content" },
+            });
+
+        private readonly IConfiguration configuration;
+
+        public HomeController(IConfiguration configuration)
+        {
+            this.configuration = configuration;
+        }
+
+        public IActionResult Index()
+        {
+            return RedirectToAction("Welcome");
+        }
+
+        [HttpGet]
+        [Route("Home/Section/Welcome")]

```

Comment:
Is there a reason we're not using the default routes (Home/Welcome etc)?

Reply:
Not really. I was following the NHS Pagination example, but it seems I was following it wrong and its backend was doing some fancy stuff. I'll remove the `Section/` and use the defaults.